recipes_source/torch_compile_caching_configuration_tutorial.rst

```diff
@@ -0,0 +1,76 @@
+Compile Time Caching Configurations
+=========================================================
+**Authors:** `Oguz Ulgen <https://github.com/oulgen>`_ and `Sam Larsen <https://github.com/masnesral>`_
+
+Introduction
+------------------
+
+PyTorch Compiler implements several caches to reduce compilation latency.
+This recipe demonstrates how you can configure various parts of the caching in ``torch.compile``.
+
+Prerequisites
+-------------------
+
+Before starting this recipe, make sure that you have the following:
+
+* Basic understanding of ``torch.compile``. See:
+
+  * `torch.compiler API documentation <https://pytorch.org/docs/stable/torch.compiler.html#torch-compiler>`__
+  * `Introduction to torch.compile <https://pytorch.org/tutorials/intermediate/torch_compile_tutorial.html>`__
+  * `Compile Time Caching in torch.compile <https://pytorch.org/tutorials/recipes/torch_compile_caching_tutorial.html>`__
+
+* PyTorch 2.4 or later
+
+Inductor Cache Settings
+----------------------------
+
+Most of these caches are in-memory, only used within the same process, and are transparent to the user. An exception is caches that store compiled FX graphs (``FXGraphCache``, ``AOTAutogradCache``). These caches allow Inductor to avoid recompilation across process boundaries when it encounters the same graph with the same Tensor input shapes (and the same configuration). The default implementation stores compiled artifacts in the system temp directory. An optional feature also supports sharing those artifacts within a cluster by storing them in a Redis database.
+
+There are a few settings relevant to caching and to FX graph caching in particular.
+The settings are accessible via environment variables listed below or can be hard-coded in Inductor’s config file.
+
+TORCHINDUCTOR_FX_GRAPH_CACHE
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+This setting enables the local FX graph cache feature, i.e., by storing artifacts in the host’s temp directory. ``1`` enables, and any other value disables it. By default, the disk location is per username, but users can enable sharing across usernames by specifying ``TORCHINDUCTOR_CACHE_DIR`` (below).
+
+TORCHINDUCTOR_AUTOGRAD_CACHE
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+This setting extends FXGraphCache to store cached results at the AOTAutograd level, instead of at the Inductor level. ``1`` enables, and any other value disables it.
```

Comment:
```suggestion
This setting extends ``FXGraphCache`` to store cached results at the ``AOTAutograd`` level, rather than at the Inductor level. Setting it to ``1`` enables this feature, while any other value disables it.
```